### PR TITLE
Add a logging statement so that in networking blips the dissapearance…

### PR DESCRIPTION
**Pull request by jayunit100**
Fixes part of seeing #1328 when it happens

### CODE REVIEW

File: pkg/manager/manager.go [hunk 1]
```diff
@@ -72,6 +73,7 @@ func New(opts Options) (Manager, error) {
 	}
 
 	// Build the controller manager.
+	klog.Infof("Making new manager from options: %v", opts)
```
Comment (randomvariable) [inline pkg/manager/manager.go:76]: This needs to be redacted. We absolutely cannot print credentials to the log.

**Review by randomvariable — CHANGES_REQUESTED** [review]

File: pkg/manager/manager.go (hunk 1 above)
Comment (jayunit100) [reply pkg/manager/manager.go]: yup, fixed

**Pull request closed without merging**